DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: bcEpiRate
 Title: Tools for Standardized Rates in Population Health Surveillance and Epidemiology
-Version: 0.1.0
+Version: 1.0.0
 Date: 2021-10-15
```

Comment:
Date needs to be updated when we make the final commit for v1.0.0

Reply:
Done